### PR TITLE
Update speaker details - Arun Michael Dsouza

### PR DESCRIPTION
**Pull request by ArunMichaelDsouza**

### CODE REVIEW

File: data/speakers.ts [hunk 1]
```diff
@@ -456,7 +456,7 @@ const speakers: Speaker[] = [
       linkedin: 'https://www.linkedin.com/in/arunmichaeldsouza/',
     },
     talks: [],
```
Comment (yakovlevyuri) [inline data/speakers.ts]: Hi @ArunMichaelDsouza , thanks for your PR! Could you please add your talks, future, and past. If you have too many of them, just last 5 for instance :) See Nicolas Hoizey as an example, please. Thank you very much again!

Comment (ArunMichaelDsouza) [reply data/speakers.ts]: Not a problem! I've updated the talks list.

Comment (yakovlevyuri) [reply data/speakers.ts]: Thank you! Everything is perfect now!

Comment (ArunMichaelDsouza) [reply data/speakers.ts]: Great!

**Pull request merged**